app/src/main/java/ke/co/svs/mykrypto/domain/model/Crypto.kt

```diff
@@ -7,5 +7,12 @@ import androidx.room.PrimaryKey
 class Crypto (
     @PrimaryKey
     val id: String,
-    val name: String,
-    val symbol: String)
\ No newline at end of file
+    val name: String?,
```

Comment:
There is a reason why there was Cypto and Crypto details. Crypto would be used in the dialog to add cryptos and CryptoDetails to show details of users crypto portfolio. Unless there is a different angle that I do not understand. There is a description in the parent ticket that explains why.